maintainers/team-list.nix

```diff
@@ -950,6 +950,15 @@ with lib.maintainers;
     shortName = "Serokell employees";
   };
 
+  stdenv = {
+    # TODO: https://github.com/NixOS/nixpkgs/issues/347955
+    members = [ ];
```

Comment:
I'm game to be in it.

Others that I would like in it :-)

- @tie
- @wolfgangwalther 
- @emilazy 
- @Ericson2314

Reply:
Getting @emilazy & yourself (@philiptaron) right now along with @reckenrode. Awaiting a response from @Ericson2314 but I am pretty sure he'll want to be on it.